lib/improver/tests/set_up_test_cubes.py

```diff
@@ -75,7 +75,7 @@ def construct_xy_coords(ypoints, xpoints, spatial_grid):
         x_coord = DimCoord(
             np.linspace(-20., 20., xpoints, dtype=np.float32),
             "longitude", units="degrees", coord_system=GLOBAL_GRID_CCRS)
-    elif 'equalarea':
+    elif spatial_grid == 'equalarea':
```

Comment:
Yep, this is what we want.  Anything calling these setup functions should be using `spatial_grid='latlon'` or `spatial_grid='equalarea'`.